package.json

```diff
@@ -77,6 +80,8 @@
     "@nuxtjs/imagemin": "^0.2.0",
     "@nuxtjs/stylelint-module": "^4.0.0",
     "@nuxtjs/vuetify": "^1.11.3",
+    "@types/cookie": "^0.4.0",
```

Comment:
Those shouldn't be added to tsconfig as well?

Reply:
It'll detect everything in @types automatically.

Reply:
Precision: when importing the package as modules.

If it's global stuff you're never importing (Like MediaSessions) then you need it in tsconfig.